notes-service/src/main/java/org/exoplatform/wiki/service/rest/NotesRestService.java

```diff
@@ -114,11 +114,11 @@ public NotesRestService(NoteService noteService,
     cc.setNoStore(true);
   }
 
-  @GET
+  @PUT
```

Comment:
Why did you change this one ? By reading the function, it seems to be a simple read (getNote)